app/konnect-reference-platform/faq.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Konnect Reference Platform - FAQ
+content_type: reference
+layout: reference
+
+products:
+    - api-ops
+works_on:
+  - konnect
+
+description: |
+    Provides a complete guide for platform builders to integrate {{site.konnect_product_name}} into 
+    their engineering organization and API delivery process. 
+
+breadcrumbs:
+  - /konnect-reference-platform/
+
+related_resources:
+- text: Reference Platform - Home
+  url: /konnect-reference-platform/
+- text: Reference Platform - FAQ
+  url: /konnect-reference-platform/faq/
+- text: Reference Platform - Kong Air
+  url: /konnect-reference-platform/kong-air/
+- text: Reference Platform - How-To
+  url: /konnect-reference-platform/how-to/
+- text: Reference Platform - APIOps
+  url: /konnect-reference-platform/apiops/
+---
+
+## What is the Konnect Reference Platform?
+The [Konnect Reference Platform](/konnect-reference-platform/) is a technical guide for users looking to integrate 
+the [{{site.konnect_product_name}}](https://konghq.com/products/kong-konnect) API Management platform into their API delivery process and broader 
+engineering organization’s technology stack.
+
+## Who can benefit from the reference platform?
+The Reference Platform is designed for software Architects, API platform builders, DevOps engineers, and Service Application 
+teams who are looking to implement a Federated API Management platform with Konnect.
+
+## What are the components of the Reference Platform?
+<u>Konnect reference usage</u>: 
```

Comment:
The underline is not really a style we use. It usually denotes links.

I'd convert this question into a table like you have on the landing page, and make each currently underlined term a link.